pom.xml

```diff
@@ -73,6 +73,11 @@
       <artifactId>ssh-credentials</artifactId>
       <version>1.6.1</version>
     </dependency>
+    <dependency>
```

Comment:
This should probably be removed so a separate change adds it. Also this plugin has not been released yet I doint think.